src/parser/options.ts

```diff
@@ -67,6 +70,8 @@ export interface Options {
    */
   operatorMapping: Map<symbol, string>;
 
+  allowCrossTypeParsing: boolean;
```

Comment:
as per this comment: https://github.com/briza-insurance/illogical/pull/137/files#r574117037

`strictTypeComparison: boolean`  ?

Add let's add documentation, this is a public lib.

Reply:
And this should documented in the `readme.md` and you have to also regenerate the documentation `npm run docs`

Reply:
9b2e2c0, 6594c76

Reply:
`npm run docs` results in an error 

- `Error: Tried to set an option (mode) that was not declared.`
- `Error: Tried to set an option (target) that was not declared.`

There's a known bug with `typedoc` [here](https://github.com/TypeStrong/typedoc/issues/1447) and i noticed you bumped up from `0.20.19` to `0.20.10`. Will investigate further

Reply:
Release notes [here](https://github.com/TypeStrong/typedoc/releases/tag/v0.20.0) states 
> Removed options - mode, excludeNotExported, includeDeclarations, ignoreCompilerErrors (#1403), entryPoint

Resolution: Remove `mode` and `target` in the `typedoc.json` and regenerate docs d7fdee1